encoders/firebase-encoders-json/gradle.properties

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version=17.1.1
```

Comment:
This change does not look right as the latest version on gmaven is 17.1.0.

Could you also double check the rest of them?